### PR TITLE
Bugfix/visual bugs 1st setember

### PR DESCRIPTION
**Pull request by clarkjoao**

### CODE REVIEW

File: components/transaction-stats.tsx [hunk 1]
```diff
@@ -4,14 +4,18 @@ import {TransactionStatus} from '@interfaces/enums/transaction-status';
 export default function TransactionStats({status = null}: {status: TransactionStatus}) {
   const ColorMap = {
     [TransactionStatus.pending]: `warning`,
-    [TransactionStatus.processing]: `info`,
+    [TransactionStatus.processing]: `blue-info`,
```
Comment (moshmage) [inline components/transaction-stats.tsx:7]: info is a bootstrap prefix, if we don't have the correct color for info then we should be editing _the variable of that colour_ and not adding a new info

also, the pending state is repeated

File: components/transaction-stats.tsx [hunk 2]
```diff
@@ -4,14 +4,18 @@ import {TransactionStatus} from '@interfaces/enums/transaction-status';
...
+    [TransactionStatus.pending]: `warning`,
+    [TransactionStatus.failed]: `red`,
     [TransactionStatus.completed]: `success`,
   }
 
   const StatusMap = {
-    [TransactionStatus.completed]: `Complete`,
     [TransactionStatus.pending]: `Pending`,
     [TransactionStatus.processing]: `Processing`,
+    [TransactionStatus.pending]: `Pending`,
```
Comment (moshmage) [inline components/transaction-stats.tsx:16]: pending is repeated

File: styles/styles.scss [hunk 3]
```diff
@@ -212,6 +212,21 @@ p {
 .text-blue-dark{
   color: $blue-dark;
 }
+.text-blue-info{
```
Comment (moshmage) [inline styles/styles.scss:215]: override the bootstrap $theme variables instead of adding new classes

File: styles/styles.scss [hunk 4]
```diff
@@ -651,8 +669,13 @@ label {
 .color-red {
   color: $red;
 }
+
+.color-orange {
```
Comment (moshmage) [inline styles/styles.scss:673]: add this to bootstraps utilities of `text-[colorname]`

File: styles/toasts.scss [hunk 5]
```diff
@@ -41,7 +41,8 @@
         justify-content: center;
         margin-right: .15rem;
         &::before {
-          content: "\1F5D9";
+          // content: "\1F5D9";
```
Comment (moshmage) [inline styles/toasts.scss:44]: ?!?!?!

**Review by moshmage — CHANGES_REQUESTED** [review]

File: next-env.d.ts [hunk 6]
```diff
@@ -1,3 +1,6 @@
 /// <reference types="next" />
 /// <reference types="next/types/global" />
 /// <reference types="next/image-types/global" />
+
+// NOTE: This file should not be edited
+// see https://nextjs.org/docs/basic-features/typescript for more information.
```
Comment (moshmage) [inline next-env.d.ts:6]: ... This link explains nothing about why this file can't be edited and thus this comments adds no real information. no need for the comment to exist

Comment (clarkjoao) [reply next-env.d.ts:6]: when I run the project, this line is always added... I will remove

File: styles/styles.scss [hunk 7]
```diff
@@ -212,6 +212,22 @@ p {
...
+}
+
+.text-red {
+  color: $red;
+}
+
+.text-orange {
+  color: $orange;
+}
+.text-success {
```
Comment (moshmage) [inline styles/styles.scss:227]: replace this values on the bootstrap variables

File: styles/styles.scss [hunk 8]
```diff
@@ -212,6 +212,22 @@ p {
...
+.text-info{
+  // !important to overwrite bootstrap css.
+  color: $blue-info !important;
+}
+
+.text-red {
+  color: $red;
+}
+
+.text-orange {
```
Comment (moshmage) [inline styles/styles.scss:224]: add this to text color utilities

File: styles/styles.scss [hunk 9]
```diff
@@ -212,6 +212,22 @@ p {
 .text-blue-dark{
   color: $blue-dark;
 }
+.text-info{
+  // !important to overwrite bootstrap css.
+  color: $blue-info !important;
+}
+
+.text-red {
```
Comment (moshmage) [inline styles/styles.scss:220]: use text-danger on the element

File: styles/styles.scss [hunk 10]
```diff
@@ -212,6 +212,22 @@ p {
 .text-blue-dark{
   color: $blue-dark;
 }
+.text-info{
+  // !important to overwrite bootstrap css.
+  color: $blue-info !important;
```
Comment (moshmage) [inline styles/styles.scss:217]: change the value on the bootstraps variables

**Review by moshmage — CHANGES_REQUESTED** [review]

**Pull request merged**